internal/show/brokers.go

```diff
@@ -51,14 +51,19 @@ func Brokers(clusterInfo *cluster.Info, status reporter.Interface) bool {
 		return true
 	}
 
-	fmt.Printf(template, "NAMESPACE", "NAME", "COMPONENTS")
+	fmt.Printf(template, "NAMESPACE", "NAME", "COMPONENTS", "DEFAULT GLOBALNET CLUSTER SIZE", "GLOBALNET CIDR RANGE", "GLOBALNET ENABLED",
+		"DEFAULT CUSTOM DOMAINS")
```

Comment:
I suggest we put the more useful columns first.

```suggestion
	fmt.Printf(template, "NAMESPACE", "NAME", "COMPONENTS", "GLOBALNET ENABLED", "GLOBALNET CIDR RANGE", "DEFAULT GLOBALNET CLUSTER SIZE", "DEFAULT CUSTOM DOMAINS")
```

Reply:
```suggestion
	fmt.Printf(template, "NAMESPACE", "NAME", "COMPONENTS", "GLOBALNET", "GLOBALNET CIDR", "DEFAULT GLOBALNET SIZE", "DEFAULT DOMAINS")
```

I suggest we also drop some superfluous terms from the header to make it more compact (and update the format accordingly).

Reply:
> I suggest we also drop some superfluous terms from the header to make it more compact (and update the format accordingly).

These are very confusing for me to understand. But I don't have a strong opinion about this. So I don't mind changing except `GLOBALNET ENABLED` because its value is bool. And showing `GLOBALNET ENABLED True` makes more sense.

Reply:
We could easily show yes/no instead of true/false which is also more user friendly but OK